recipe/meta.yaml

```diff
@@ -90,7 +90,6 @@ outputs:
         - numpy
         - scipy
         - scikit-learn
-        - setuptools
```

Comment:
It turns out this was not correct:

https://github.com/dmlc/xgboost/blob/v1.5.1/python-package/xgboost/compat.py#L104-L111

`pkg_resources` (which is shipped with `setuptools`) must be imported successfully for this code to work. It's a bit deceiving because the `import` statement was placed inside the `try/except` block, which usually suggests an optional import. But in fact, the `except` statement depends upon the successful import of `pkg_resources`! Not the best code design.